test/1-basic-test.js

```diff
@@ -1573,6 +1573,16 @@ describe('XLSX_CALC', function() {
         });
     });
 
+    describe('The Silence of Errors', () => {
```

Comment:
@LudaRu here is a code that shows how to ignore errors and keep logging or do all the stuff silent